packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -137,6 +156,94 @@ export interface PluginConfigTypeIos {
    * SDK 49+ is required.
```

Comment:
```suggestion
   * SDK 49 or higher is required.
```

Unrelated to the changes in this PR but above suggestion adds word consistency on how we frame when talking about a specific and higher SDK versions.

Reply:
thanks but i will keep this as-is. since the doc is versioned and i will remove the **SDK 49 is required** soon in a pr to rename `unstable_networkInspector` to `networkInspector`.